### PR TITLE
Create BLorient13269

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I created this record according to S. Strelcyns catalogue. 

I was not quite sure whether LIT6563HassabHeg was the correct ID for Ḥasāba ḥǝgg (movable feast calendar) in ms_i1. I think it is the ID for all calendar calculations for movable festivals, no matter if these calculations start and end in different years and use different cycles. 

I was even less sure about ms_i2. Strelcyn refers to it simply as "Kidan", as was the case in BLorient8813. But here there is no incipit or other information, so I can only speculate whether it is LIT2260salota again, or LIT1938Mashaf, or some other work with that name. I think it would be better to leave it unmarked.

### CODE REVIEW

**Comment by DenisNosnitsin1970** [comment]
The Hassabe heg with 6563 appears to be different, as to Kidan, please check LIT3058RepCh338.

**Comment by CarstenHoffmannMarburg** [comment]
> The Hassabe heg with 6563 appears to be different,

The incipit in BLorient13269 seems to be more close to the one attested in IVorlov34, that does not have a markup, yet:

```
<item xml:id="a2">
                           <desc type="Unclear" xml:lang="gez"/>
                          <locus from="2v" to="3v"/>
                          <q xml:lang="gez"> በዝ᎓ ንዜከር᎓  ሐሳበ᎓ ሕጉ᎓ ለእግዚእነ᎓ ወመድኃኒነ᎓ ኢ᎓ ክ᎓
                            <gap reason="omitted" resp="PRS9602Turaev"/>እንዘ᎓ ሀሎነ᎓ በዘበነ᎓ ቅዱስ᎓ ማርቆስ᎓
                            ወንገላዊ᎓ ይብጽሐነ᎓ እስከ᎓ ዘበነ᎓ ሉቃስ᎓ ዜናዊ። በ፶፲ት᎓ በ፭ቱ፻ት᎓ ኮነ᎓ ዓመተ᎓ ዓለም። በ፲ቱ᎓ ፻።
                            በ፲ቱ᎓ ፻፺᎓ ኮነ᎓ ዓመተ᎓ ምሕረት᎓<gap reason="omitted" resp="PRS9602Turaev"/>
                           </q>
                           <note>Chronological notes written on a sewn-in European paper leaf.</note>
                       </item>
```
Do you think, it is appropriate to create a new ID in this case?

**Comment by CarstenHoffmannMarburg** [comment]
> as to Kidan, please check LIT3058RepCh338.

I have used LIT3058RepCh338 for ms_i3 as it is evident from the incipit.

However, for ms_i2, I have no information beyond the title kidān.

Screenshot 2024_Kidan

**Comment by DenisNosnitsin1970** [comment]
Yes, it is upropriate to make a new record for Hassaba heg, you can use relations to disambiguate it from  LIT6563HassabHeg. Please lead a discussion in "Issues", here we only correct / comment upon descriptions, for better transparency.

File: LondonBritishLibrary/orient/BLorient13269.xml [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianContent"/>
+                    <term key="ChristianLiterature"/>
+                </keywords>
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient13269.xml:199]: Chronography
Poetry
Liturgy

File: LondonBritishLibrary/orient/BLorient13269.xml [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+                            <handNote script="Ethiopic" xml:id="h3">
+                                <desc>More regular and larger than the others with a height of about 4mm.</desc>
+                                <date notBefore="1600" notAfter="1700" resp="PRS8999Strelcyn"/>
+                            </handNote>
+                        </handDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="#73v"/>
+                                    <desc type="GuestText">Traces of the beginning of a <foreign xml:lang="gez">salām</foreign> to the Archangel
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient13269.xml:152]: we also have PoemSalam as a type for additions

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Review by eu-genia — APPROVED** [review]

**Pull request merged**